### DISCUSSION TITLE
Error Saving

### DISCUSSION THREAD

**Discussion by azaleob** [givanz/Vvveb#128] [Q&A]
Hello,

I just recently installed vvveb, I like this CMS from all what I have tested, after the installation I just tried to Edit Website | Homepage and in the navigation links of the Homepage, I just rename About to Test and then click save, but it  gives me Error Saving. 

How do I fix this?

Regards
Azaleo

**Comment by givanz** [comment]
Hi

Save error happens only after page rename? or it happens on every occasion?

If save does not work at all please check if  theme folder is writable `/public/themes/landing`  replace `landing` if you use another theme. 

If saving does not work only after rename, there was a recent fix for page rename https://github.com/givanz/Vvveb/commit/be66cf1807633de1ceef7212f37edec897daf00e 

Can you please redownload the zip and update the files on the server and see if it fixes the bug?

You can skip `/config` folder update to keep the database info and avoid reinstall and also skip `/public/themes` if you want to keep your layout changes. 

If still doesn't work you can check for error messages in `admin > tools > system info > error log` and provide the error message if any and I can further investigate.

https://demo.vvveb.com/admin/index.php?module=tools/errorlog
